Cargo.toml

```diff
@@ -58,8 +58,9 @@ rsa-der = "0.2"
 rsa-export = "0.2"
 # avoid duplicate dependency by sticking to 0.1
 oid = "0.1"
-picky = "6"
-picky-asn1-x509 = "0.3"
+# Pin minor version to make sure we use the exact same picky-asn1-x509 version
+picky = "~6.2"
+picky-asn1-x509 = "0.5"
```

Comment:
Should we not pin the whole picky crates then?

Reply:
I'm somewhat hoping that patch bumps to `6.2` would not include a minor bump to `picky-asn1-x509` - this seems to be a breaking change since the latter is part of its public API but I don't think `picky` gives any guarantee about that.

Do you want me to pin `picky` to exact/patch-level version then? If so, should I also pin the x509 crate?

Reply:
I think we should pin them all to fully specified numbers where we have cross-dependencies, so yes, I think that's the best path to avoid future breakage. Thanks!

Reply:
Updated :)